src/modules/actions/requestFilmsByGenre.js

```diff
@@ -0,0 +1,44 @@
+import { GET_FILMS, API_KEY, BASE_URL } from '../constants/constants';
+import showListFilms from './showListFilms';
+
+let arr = [];
+
+const getFilms = (results) => ({
+  type: GET_FILMS,
+  payload: results
+})
+
+const findId = (genres, genreFilm) => {
+  const result = genres.filter(item => item.name === genreFilm)
+  return result[0]?.id;
+}
+
+const sortFilmsByGenre = (arrFilms, id) => {
```

Comment:
this function should be like a helper
it doesn't need to know about some array outside

let it take one array as argument and return each time new one 

`const sortFilmsByGenre = (arrFilms, id) => arrFilms.filter(item => item.genre_ids.includes(id));
`